tests/functional/h/views/admin/test_permissions.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+
+class TestAdminPermissions:
+    PAGES = (
+        # URL, accessible by role.Staff?
+        ("/admin/", True),
+        ("/admin/admins", False),
+        ("/admin/badge", False),
+        ("/admin/features", False),
+        ("/admin/groups", True),
+        ("/admin/mailer", True),
+        ("/admin/nipsa", False),
+        ("/admin/oauthclients", False),
+        ("/admin/organizations", True),
+        ("/admin/staff", False),
+        ("/admin/users", True),
+        ("/admin/search", False),
```

Comment:
It's not mega obvious but if you look through the code in the admin pages, you always need the same permission to view the page as you do to take actions on it.

So if you test you can see the page, you've indirectly tested you have the right permission to do the thing.